pkg/clusterversion/cv.go

```diff
@@ -0,0 +1,58 @@
+package clusterversion
+
+import (
+	"context"
+
+	configv1 "github.com/openshift/api/config/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+var (
+	OSD_CV_NAME string = "version"
```

Comment:
Should use camel format here?
And looks like this variable is not being used in here. Should it be declared somewhere else?
Another question is this should be a `const` or `var`?

Reply:
Yeah, im in 2 minds about the way we do this. I'll remove it for now. Thanks @bmeng